src/embed_tests/TestPythonException.cs

```diff
@@ -54,5 +54,41 @@ public void TestPythonErrorTypeName()
                 Assert.That(ex.PythonTypeName, Is.EqualTo("ModuleNotFoundError").Or.EqualTo("ImportError"));
             }
         }
+
+        [Test]
+        public void TestPythonExceptionFormat()
+        {
+            try
+            {
+                PythonEngine.Exec("raise ValueError('Error!')");
+                Assert.Fail("Exception should have been raised");
+            }
+            catch (PythonException ex)
+            {
+                Assert.That(ex.Format(), Does.Contain("Traceback").And.Contains("(most recent call last):").And.Contains("ValueError: Error!"));
```

Comment:
Not sure if Python exceptions messages are localized. If that is the case, you need to ensure the right locale is used before performing this test.

Reply:
I will try and figure out if exception messages are localized.